src/views/Room/CustomTimerDialog/index.jsx

```
@@ -0,0 +1,79 @@
+} from '@mui/material';
+import { ChangeCircle } from '@mui/icons-material';
+import { useTranslation } from 'react-i18next';
+import { useState } from 'react';
+
+const CustomTimerDialog = ({ isOpen, onClose, onSubmit }) => {
+  const { t } = useTranslation();
+  const [customTime, setCustomTime] = useState('');
+  const [isMinutes, setIsMinutes] = useState(false);
+
+  const handleSubmit = () => {
+    let time = parseInt(customTime, 10);
```

Biome

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)